packages/core/networks/1116.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "contractName": "Registry",
```

Comment:
I went through all these, spot checked ownership/fields and checked verifications. All looked good!